seqr/views/utils/variant_utils.py

```diff
@@ -131,6 +131,20 @@ def _get_rna_seq_outliers(gene_ids, families):
     return data_by_individual_gene
 
 
+def _get_phenotype_prioritization(gene_ids, families):
+    data_by_individual_gene = defaultdict(lambda: defaultdict(lambda: defaultdict(list)))
+
+    data_dicts = _get_json_for_models(
```

Comment:
Is there a reason you defined a `get_json_for_phenotype_prioritization` helper function and don't use it here?

Reply:
The only thing that the `get_json_for_phenotype_prioritization` does is formating the score field names. I think we can get rid of the helper function in the end.